forum/qa-plugin/report-reason/script.js

```diff
@@ -36,31 +55,35 @@ document.addEventListener("DOMContentLoaded", () => {
                  cache: false,
                  success: function(data)
                  {
-                    if(data.error)
-                    {
-                        errorText = errorMessage;
-                    }
-                    else if(data.success)
-                    {
-                        location.reload();
-                    }
-                    else
-                    {
-			            errorText = errorMessage;
-                    }
+                     if(data.error)
+                     {
+                         errorText = errorMessage;
+                     }
+                     else if(data.success)
+                     {
+                         location.reload();
+                     }
+                     else
+                     {
+	                 errorText = errorMessage;
+                     }
                  },
                  error: function(data)
                  {
-		             errorText = errorMessage;
+	             errorText = errorMessage; 
                  }
-            });
-	
+            });	
+        } else {
+            errorText = "Proszę wybrać powód zgłoszenia!";    
+        }
         if(errorText)
-	    {
-		    const errorHtml = document.createElement('p');
-		    errorHtml.innerHTML = errorText;
-		    errorPopupHtml.appendChild(errorHtml);
-	    }
+        {
+            const errorPopupHtml = document.querySelector('.html-error');
+	    const errorHtml = document.createElement('p');
+            errorHtml.innerHTML = errorText;
```

Comment:
Jeśli tekst jakiegoś elementu nie jest HTMLem, lecz zwykłym tekstem, to nie ma potrzeby [uruchamiać parsera HTML](https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#Operational_details) - wystarczy użyć `textContent`.